src/src/rx-serial/SerialMavlink.cpp

```diff
@@ -50,13 +51,13 @@ void SerialMavlink::sendQueuedData(uint32_t maxBytesToSend)
 SerialMavlink::SerialMavlink(Stream &out, Stream &in):
     SerialIO(&out, &in),
     // 255 is typically used by the GCS, for RC override to work in ArduPilot `SYSID_MYGCS` must be set to this value (255 is the default)
-    this_system_id(255),
+    this_system_id(config.GetSourceSysId()?config.GetSourceSysId():255),
     // Strictly this is not a valid source component ID
-    this_component_id(MAV_COMPONENT::MAV_COMP_ID_ALL),
+    this_component_id(MAV_COMPONENT::MAV_COMP_ID_TELEMETRY_RADIO), //telemetry radio fits better
     // Assume vehicle system ID is 1, ArduPilot's `SYSID_THISMAV` parameter. (1 is the default)
-    target_system_id(1),
+    target_system_id(config.GetTargetSysId()?config.GetTargetSysId():1),
```

Comment:
Definitely need some whitespace here plz:
`target_system_id(config.GetTargetSysId() ? config.GetTargetSysId() : 1),`

Reply:
Thats fair, I added all the whitespace :-) and clarified comments